app/views/sign_up/select_email/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<% self.title = t('titles.select_email') %>
+
+<%= render StatusPageComponent.new(status: :info, icon: :question) do |c| %>
+  <% c.with_header { t('titles.select_email') } %>
+  <p class="margin-top-4">
+    <%= I18n.t('help_text.select_preferred_email', sp: @sp_name, app_name: APP_NAME) %>
+  </p>
+
+  <%= simple_form_for('', url: sign_up_select_email_path) do |f| %>
+    <% @user_emails.each do |email, index| %>
+      <fieldset class="usa-fieldset">
+        <div class="usa-radio">
+          <%= radio_button_tag(
+                'select_email_form[selected_email_id]',
+                email.id,
+                email.email == @last_sign_in_email_address,
+                class: 'usa-radio__input usa-radio__input--bordered',
+              ) %>
+              <%= label_tag(
+                    "select_email_form_selected_email_id_#{email.id}",
+                    class: 'usa-radio__label width-full',
+                  ) do %>
+                <%= email.email %>
+              <% end %>
+        </div>
+      </fieldset>
+    <% end %>
+    <%= f.submit t('help_text.requested_attributes.change_email_link'), class: 'margin-top-4' %>
+  <% end %>
+
+    <%= render ButtonComponent.new(
+          url: add_email_path,
+          outline: true,
+          big: true,
+          wide: true,
+          class: 'margin-y-2',
```

Comment:
Having the margin on the bottom creates too big a gap between the button and the divider at the bottom of the page, per the designs.

```suggestion
          class: 'margin-top-2',
```